src/IROperator.h

```diff
@@ -1257,6 +1257,11 @@ inline Expr erf(Expr x) {
     return Internal::halide_erf(std::move(x));
 }
 
+/** Fast vectorizable approximation to some trigonometric functions for Float(32).
+ * Absolute approximation error is less than 1e-5. */
+Expr fast_sin(Expr x);
```

Comment:
To get doxygen comments to work right for groups of functions you need to surround them with // @{ and // @}